components/form/introduction.rst

```diff
@@ -556,7 +556,7 @@ method:
 
         $request = Request::createFromGlobals();
 
-        $form->handleRequest($request);
+        $form->handleRequest();
```

Comment:
As a `Request` instance is created in this example, I guess the fix should be to add the `HttpFoundation` integration before those lines and revert this change:
```php
use Symfony\Component\Form\Forms;
use Symfony\Component\Form\Extension\HttpFoundation\HttpFoundationExtension;

$formFactory = Forms::createFormFactoryBuilder()
    ->addExtension(new HttpFoundationExtension())
    ->getFormFactory();
```

This will add the `HttpFoundationRequestHandler`, instead of using the base `NativeRequestHandler`.